Support/Scripts/run-lldb-tests.sh

```diff
@@ -25,12 +25,14 @@ source "$top/Support/Scripts/common.sh"
 [ -x "$build_dir/ds2" ]   || die "Unable to find a ds2 binary in the current directory."
 
 opt_fast=false
+opt_no_ds2_blacklists=false
```

Comment:
In this case maybe rename the argument from `--no-blacklists` to `--no-ds2-blacklists`? I agree that it's useful to specify that we're talking about the ds2 blacklists only here.